wpimath/src/main/native/include/frc/kinematics/SwerveDriveKinematics.inc

```diff
@@ -66,7 +66,8 @@ SwerveDriveKinematics<NumModules>::ToSwerveModuleStates(
     units::meters_per_second_t y{moduleStateMatrix(i * 2 + 1, 0)};
 
     auto speed = units::math::hypot(x, y);
-    Rotation2d rotation{x.value(), y.value()};
+    auto rotation = speed.value() > 1e-6 ? Rotation2d{x.value(), y.value()}
```

Comment:
```suggestion
    auto rotation = units::math::abs(speed) > 1e-6_mps ? Rotation2d{x.value(), y.value()}
```

Reply:
Ditto, though I can make the switch to proper usage of units